docs/Building.md

```diff
@@ -140,3 +142,52 @@ Build Release
 > make
 ```
 
+## Building the VIP files for LV Support, LV Servicer and Client and Server templates
+
+This section covers building the VIP files from the labview source so that you can install them for a supported version of LabVIEW (2019 or Higher).
+
+You would need to build the VIP files if you make changes to the LabVIEW pieces of the grpc-labview repository.
+
+** Building of VIP files using the instructions below is only supported on Windows **
+
+### Prerequisites
+
+- Install LabVIEW 2019 32 bit
+- Install LabVIEW CLI
+- Install JKI VI Package Manager. You would need Community or Pro edition to build VIP files.
+- Install VI Package Manager API for LabVIEW. (Download [location](https://www.ni.com/en-in/support/downloads/tools-network/download.vi-package-manager-api.html#374501))
+- Install Python 3.7 or Higher
+
+** Apart from these you would need the Prerequisites for building the binaries as well if you use the --buildcpp option described below. **
+
+### Building VIP files
+
+Download the repo. Run the command below to build the VIP files.
+
+```
+> python build-it\build.py --target <BUILD_TARGET> [--pathToBinaries <PATH_TO_PREBUILT_BINARIES>] [--buildcpp]
+```
+
+The accepted vaues for BUILD_TARGET are:
+- Win32
+- Win64
+- All
+
+If you choose to build for Win32 or Win64 target you can specify the *--buildcpp* option to indicate that you want to build the C++ binaries first and then use them to build VIP files. If you don't specify this option it is assumed that the required binaries are already built accoriding to the steps described above.
+
+If you choose to build "All" target then you need to specify a folder where we can find the pre built binaries for all the supported targets using the *--pathToBinaries** option. We expect the folder specified to have the following structure.
```

Comment:
Lets just make it clear here that with --All , even if one gives --buildcpp it just ignores it and need --pathToBinaries

Reply:
Done.